### ISSUE TITLE
[2.6 RCx] Bug: certain pump events are not logged in AAPS

### ISSUE THREAD

**Issue by sailor-911** [MilosKozak/AndroidAPS#2434]
**Pump used**: Roche insight
**AAPS versions affected**: since 2.6 RC4 until latest version RC7, probably all 2.6RCx
**Event types **not** recorded**: reservoir change, batterie change, cannula change, katheter change

**Remark**: in last master 2.5.1 it was correct

**Workaround**: to record these events directly in AAPS via actions dialog

**Comment by MilosKozak** [comment]
log?

**Comment by sailor-911** [comment]
today I had a reservoir change - but all other event types too (see issue description) triggered by the insight pump are not recorded. Change was today around 16:00. Logs added.

[AndroidAPS._2020-02-06_12-17-40_.10.zip](https://github.com/MilosKozak/AndroidAPS/files/4167533/AndroidAPS._2020-02-06_12-17-40_.10.zip)
[AndroidAPS._2020-02-06_12-17-40_.11.zip](https://github.com/MilosKozak/AndroidAPS/files/4167534/AndroidAPS._2020-02-06_12-17-40_.11.zip)

**Comment by MilosKozak** [comment]
log ends 15.50

**Comment by sailor-911** [comment]
mhh, ok. here’s the successor with id12. hope this does match

[AndroidAPS._2020-02-06_12-17-40_.12.zip](https://github.com/MilosKozak/AndroidAPS/files/4167871/AndroidAPS._2020-02-06_12-17-40_.12.zip)

**Comment by TebbeUbben** [comment]
Could you try unchecking and re-checking all the respective settings?

**Comment by sailor-911** [comment]
Pump: insight

0. Switched OFF all options to record pump events in AAPS; did an AAPS restart; switched ON again all options for pump events
1. did a pump de-connect
2. startet a cannula exchange @10:05 and did perform the fill up by the pump’s option (at first not in AAPS).
3. Afterwards I did a pump reconnect to get all the events recorded in AAPS, but no event was shown in graph / careportal
4. @10:07 I did a record in AAPS via careportal
5. —> no pump events (see description) are recorded, only records set up in AAPS are found in careportal.
**Remark**: 
Status information is working: battery load level, reservoir load level

log is attached
[AndroidAPS.log](https://github.com/MilosKozak/AndroidAPS/files/4169940/AndroidAPS.log)
